### PR TITLE
Products

### PR DESCRIPTION
**Pull request by Digambar1411**
feat : added products filters to products

1. filter by price range
2. sort by price
3. filter by brands
4. filter by ratings
clear filter is not added will implement that in next PR

Preview link: https://deploy-preview-4--lustrous-wisp-19f9e7.netlify.app/

### CODE REVIEW

File: src/frontend/pages/products/products.jsx [hunk 1]
```diff
@@ -1,8 +1,52 @@
...
+    const {state } = useFilter();
+
+    const {sortPrice, priceRange, brands,ratings} = state;
+
+    const sortedProducts = sortBypriceFunc(data,sortPrice); 
+    const priceRangeFilteredProducts = filterByPriceRangeFunc(sortedProducts, priceRange); 
+    const brandFilterProducts = filterByBrandFunc(priceRangeFilteredProducts, brands) 
+    const finalProducts = filterByRatingFunc(brandFilterProducts, ratings)
+    
+    // console.log(finalProducts);
```
Comment (tushargoyal812) [inline src/frontend/pages/products/products.jsx]: remove comments

**Review by tushargoyal812 — COMMENTED** [review]
code looks clean and good.
variable naming is good.
folder structure is good.

**Comment by horsemaker** [comment]
image

Initially, the user can't see the price range.

**Review by horsemaker — APPROVED** [review]
Looks good Digambar!

**Comment by Digambar1411** [comment]
> image
> 
> Initially, the user can't see the price range.

yeah thanks for pointing out i have fixed that

**Comment by Digambar1411** [comment]
> code looks clean and good. variable naming is good. folder structure is good.

thanks tushar

**Pull request merged**